src/registration.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+
+<html>
+    <head>
+    </head>
+    <body>
+        <header>
+            vinyl exchange
+        </header>
+        <main>
+            <form>
+                <div>
+                    <label>
+                        First Name <input id="firstname" type="text"/>
+                    </label>
+                </div>
+                <div>
+                    <label>
+                        Last Name <input id="lastname" type="text"/>
+                    </label>
+                </div>
+                <div>
+                    <label>
+                        Male <input type="radio" name="gender" id="male" value="0">
+                        Female <input type="radio" name="gender" id="female" value="1">
+                    </label>
+                </div>
+                <div>
+                    <label>
+                        Date of Birth <input id="dateofbirth" type="date"/>
+                    </label>
+                </div>
+                <div>
+                    <label>
+                        Email Address <input id="email" type="text"/>
```

Comment:
There is a specific `type` value for email fields, surprisingly called `email`. With that, your browser will enforce a typical email address format for that input field.
Check out this documentation on the various types of... `type`s: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input

- [ ] Change type attribute to `email`.

Reply:
Done, text type changed to email. :)